components/gitpod-protocol/src/teams-projects-protocol.ts

```diff
@@ -117,6 +117,11 @@ export interface Team {
     deleted?: boolean;
 }
 
+export interface InstallationAdmin {
```

Comment:
I wonder if the file `teams-projects-protocol.ts` is the proper place. What do you think of creating a dedicated file for this (`installation-admin-protocol.ts` or something similar)?

Reply:
Yeah, it probably isn't the correct place at all. I think when I was writing it, I misunderstood that file as "HERE ARE ALL THE THINGS"